src/Compilers/Core/Portable/Diagnostic/DiagnosticSeverity.cs

```diff
@@ -27,7 +27,12 @@ public enum DiagnosticSeverity
         /// <summary>
         /// Something not allowed by the rules of the language or other authority.
         /// </summary>
-        Error = 3
+        Error = 3,
+
+        /// <summary>
+        /// Something that can be potentially improved.
+        /// </summary>
+        Suggestion = 4,
```

Comment:
I agree with Sri, this seems to just complicate the severity kinds. Its hard to distinguish when a diagnostic should be a suggestion or an info. I like the way error list buckets them: errors, warnings and messages (obviously it doesn't have hidden that we need for severity). Suggestion and Info both seem like information messages about your code.